scripts/sysman/EXT_Exp_FirmwareSecurityVersion.rst

```diff
@@ -0,0 +1,30 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_ras_state:
+
+======================================
+ Firmware security firmware version
+=====================================
+
+API
+----
+
+* Functions
+
+    * ${s}FirmwareGetSecurityVersionExp
+    * ${s}FirmwareSetSecurityVersionExp
+
+* Enumerations
+
+    * ${s}_firmware_security_exp_version_t
+
+Security Version
```

Comment:
could you add more literature on objectives for this extension and programming example